infra/networks/main.tf

```diff
@@ -93,3 +93,31 @@ resource "aws_vpc_endpoint" "aws_service" {
   subnet_ids          = data.aws_subnets.default.ids
   private_dns_enabled = true
 }
+
+# VPC Configuration for DMS
+# ----------------------------------------
+#
+data "aws_ssm_parameter" "dms_peer_owner_id" {
+  name = "/network/dms/peer-owner-id"
+}
+
+data "aws_ssm_parameter" "dms_peer_vpc_id" {
+  name = "/network/dms/peer-vpc-id"
+}
+
+resource "aws_vpc_peering_connection" "dms" {
+  peer_owner_id = data.aws_ssm_parameter.dms_peer_owner_id.value
+  peer_vpc_id   = data.aws_ssm_parameter.dms_peer_vpc_id.value
+  vpc_id        = data.aws_vpc.default.id
+  peer_region   = "us-east-2"
```

Comment:
Just checking that this is the right region since us-east-1 is generally the one where I've seen things configured on the east coast.

Reply:
We are enabling inter-region VPC peering, and the `peer_region` is where MicroHealth test db lives, so this is correct and the peering connection is active

Reply:
We are enabling inter-region VPC peering, and the `peer_region` doesn't refer to our region, it is where MicroHealth test db lives, so this is correct and the peering connection is active. The peering on our end is in `us-east-1` where our infra lives.